old-faithful-proto/proto/old-faithful.proto

```diff
@@ -13,6 +14,8 @@ message VersionRequest {}
 
 message VersionResponse {
   string version = 1;
+  string commit = 2;
+  repeated uint64 epochs = 3; 
```

Comment:
while it's nice to have a field per value we can put json serialized data to the version field and add fields without changing proto :melting_face:

Reply:
sounds good

Reply:
This an example what I use in grpc for version: https://github.com/rpcpool/yellowstone-grpc/blob/v1.14.4%2Bsolana.1.18.15/yellowstone-grpc-geyser/src/version.rs#L30